source/2020-04-24-the-ember-times-issue-145.md

```diff
@@ -0,0 +1,130 @@
+---
+title: The Ember Times - Issue No. 145
+author: Chris Ng, Matthew Roderick, Jared Galanis, Isaac Lee, Jessica Jordan, Amy Lam
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
+alias : "blog/2020/04/24-the-ember-times-issue-145.html"
+responsive: true
+---
+
+👋 Emberistas! 🐹
+
+Ember Styleguide 4.0.0 release 🔥,
+autotracking case study with TrackedMap 🐾🗺,
+simple drag and drop in Ember Octane 🐺,
+meet 2 developers who became Ember developers 🎉,
+and last, but not least, watch the new Ember Map video on the `{{fn}}` helper 📺!
```

Comment:
```suggestion
and last, but not least, watch the new EmberMap video on the `{{fn}}` helper 📺!
```

Reply:
sorry just saw this

feel free to @ me on discord next time! 

i'll make this change, already made in goodbits 

this would also be good for @roderickdevelopers to fix in old issues if interested, could practice making a PR on your own. matt, what do you think?

![image](https://user-images.githubusercontent.com/1372946/80265955-4ca9f800-864e-11ea-961f-71ad36e526c1.png)

Reply:
will DM on discord

Reply:
Yep, I like the idea of asking Matthew to help!